pkg/products/rhssocommon/reconciler_test.go

```diff
@@ -1114,18 +1114,18 @@ func TestReconciler_ReconcileCSVEnvVars(t *testing.T) {
 				for _, e := range envs {
 					if e.Name == "PHILS_COOL_ENV_VAR" {
 						if e.Value != "yesboi" {
-							return errors.New(fmt.Sprintf("Expected env var {PHILS_COOL_ENV_VAR: yesboi}, got %+v", envs))
+							return fmt.Errorf("expected env var {PHILS_COOL_ENV_VAR: yesboi}, got %+v", envs)
 						}
 					} else if e.Name == "PAULS_AMAZEBALLS_ENV_VAR" {
 						if e.Value != "yeaaah" {
-							return errors.New(fmt.Sprintf("Expected env var {PAULS_AMAZEBALLS_ENV_VAR: yeaaah}, got %+v", envs))
+							return fmt.Errorf("expected env var {PAULS_AMAZEBALLS_ENV_VAR: yeaaah}, got %+v", envs)
 						}
 					} else {
-						return errors.New(fmt.Sprintf("unexpected env var (should only have PHILS_COOL_ENV_VAR and PAULS_AMAZEBALLS_ENV_VAR): %+v", envs))
+						return fmt.Errorf("unexpected env var (should only have PHILS_COOL_ENV_VAR and PAULS_AMAZEBALLS_ENV_VAR): %+v", envs)
 					}
 				}
 				if !updated {
-					return errors.New(fmt.Sprintf("Expected updated to be true, but got: %v", updated))
+					return fmt.Errorf("wxpected updated to be true, but got: %v", updated)
```

Comment:
```suggestion
					return fmt.Errorf("expected updated to be true, but got: %v", updated)
```

Reply:
Running make test/lint returns no errors. Happy to add lgtm when above is updated.

Reply:
Nice catch thank you! I just pushed the fix